### DISCUSSION TITLE
Additional space in X and Y

### DISCUSSION THREAD

**Discussion by gloomie** [highfestiva/finplot#318] [Q&A] [answered]
On a candlestick plot, how can I add:
1) blank space on X axis at right (eg +15% of the total dates) - this helps a lot on visual projection of price
2) wider range of Y axis (eg +10% from higher High and -10% from lower Low) - same reason, helps with projections in future

set_y_range didn't work for 2...

I have drawn a dummy second plot in the same ax to do this (with fake prices and dates) but can't believe it is not doable with a proper way. This functionality exists in most libraries and of course in Tradingview.  

Thanks!

**Comment by highfestiva** [comment, marked as answer]
Before plotting:
```python
fplt.right_margin_candles = 30 #  percent not available
fplt.y_pad = 0.1 # 10%
```
